prow/cmd/build/controller.go

```diff
@@ -17,46 +17,424 @@ limitations under the License.
 package main
 
 import (
-	_ "errors"
-	_ "flag"
-	_ "fmt"
-	_ "os"
-	_ "os/signal"
-	_ "reflect"
-	_ "syscall"
-	_ "time"
-
-	_ "k8s.io/test-infra/prow/apis/prowjobs/v1"
-	_ "k8s.io/test-infra/prow/client/clientset/versioned"
-	_ "k8s.io/test-infra/prow/client/clientset/versioned/scheme"
-	_ "k8s.io/test-infra/prow/client/informers/externalversions"
-	_ "k8s.io/test-infra/prow/client/informers/externalversions/prowjobs/v1"
-	_ "k8s.io/test-infra/prow/client/listers/prowjobs/v1"
-	_ "k8s.io/test-infra/prow/logrusutil"
-
-	_ "github.com/knative/build/pkg/apis/build/v1alpha1"
-	_ "github.com/knative/build/pkg/client/clientset/versioned"
-	_ "github.com/knative/build/pkg/client/informers/externalversions"
-	_ "github.com/knative/build/pkg/client/informers/externalversions/build/v1alpha1"
-	_ "github.com/knative/build/pkg/client/listers/build/v1alpha1"
-
-	_ "k8s.io/api/core/v1"
-	_ "k8s.io/apimachinery/pkg/api/errors"
-	_ "k8s.io/apimachinery/pkg/apis/meta/v1"
-	_ "k8s.io/apimachinery/pkg/runtime/schema"
-	_ "k8s.io/apimachinery/pkg/util/runtime"
-	_ "k8s.io/apimachinery/pkg/util/wait"
-	_ "k8s.io/client-go/kubernetes"
-	_ "k8s.io/client-go/kubernetes/scheme"
-	_ "k8s.io/client-go/kubernetes/typed/core/v1"
-	_ "k8s.io/client-go/tools/cache"
-	_ "k8s.io/client-go/tools/clientcmd"
-	_ "k8s.io/client-go/tools/record"
-	_ "k8s.io/client-go/util/workqueue"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"reflect"
+	"syscall"
+	"time"
+
+	prowjobv1 "k8s.io/test-infra/prow/apis/prowjobs/v1"
```

Comment:
@stevekuznetsov `goimports` strikes again? /shrug

Reply:
Doesn't look like it, @fejta chose to use `prowjobv1` in his code.

Reply:
yeah just the bulk _ -> named imports looked like that goimports change, I have no problem with this line.